packages/adaptive_navigation/lib/src/adaptive_navigation_scaffold.dart

```diff
@@ -39,12 +40,12 @@ class AdaptiveScaffoldDestination {
   });
 }
 
-/// A widget that adapts to the current display size, displaying a [Drawer],
-/// [NavigationRail], or [BottomNavigationBar]. Navigation destinations are
+/// A widget that adapts to the current display size, displaying a [Drawer]/[NavigationDrawer],
```

Comment:
please adopt https://dart.dev/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph

also, can you clarify that [NavigationDrawer] is if the ambient [ThemeData.useMaterial3] is true, otherwise [Drawer] is used.